backend/Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM node:15.0.1
 WORKDIR /foodies/backend
-COPY *.json /foodies/backend/
+COPY package.json ./
+COPY yarn.lock ./
```

Comment:
Added to copy `yarn.lock`, no need for wildcard because we may end up copying `eslintrc.json` which we do not need for `RUN yarn install`.

I also checked that the dest `./` works because it's relative to `WORKDIR`. I verified by viewing the file system in the docker container and saw that it was copied correctly